robolectric-shadows/shadows-core/src/main/resources/org/robolectric/shadows/ShadowAndroidHttpClient.java.vm

```diff
@@ -0,0 +1,93 @@
+#if ($api >= 23)
```

Comment:
`$api >= 23` ? Is this correct? It was removed in API 23 so maybe `$api < 23` or `$api <= 22` is the corrent way?